rllib/algorithms/algorithm.py

```diff
@@ -685,7 +685,6 @@ def setup(self, config: AlgorithmConfig) -> None:
                     )
                     and self.config.input_ != "sampler"
                     and self.config.enable_rl_module_and_learner
-                    and self.config.enable_env_runner_and_connector_v2
```

Comment:
note: we don 't need these double checks anymore b/c the hybrid stack has been deprecated already (users will get error message in the `config.validate()` call).

Reply:
Yes, let's remove it!